notebooks/Accessing_File_with_GDS.ipynb

```diff
@@ -34,7 +34,7 @@
     "\n",
     "or\n",
     "```\n",
-    "!pip install cupy-cuda110\n",
+    "!pip install cupy-cuda12x\n",
     "!pip install torch==1.7.1+cu110 torchvision==0.8.2+cu110 torchaudio==0.7.2 -f https://download.pytorch.org/whl/torch_stable.html\n",
```

Comment:
@grlee77 thanks for updating the CuPy packages! 🙏

Should we do the same with PyTorch and friends?

There are a bunch more lines like these throughout the notebooks. Perhaps we should update them all and rerun to make sure they are working ok